scripts/shared/clusters.sh

```diff
@@ -118,9 +136,9 @@ function create_kind_cluster() {
     export KUBECONFIG=${KUBECONFIGS_DIR}/kind-config-${cluster}
     rm -f "$KUBECONFIG"
 
-    if kind get clusters | grep -q "^${cluster}$"; then
+    if ${kind} get clusters | grep -q "^${cluster}$"; then
```

Comment:
```suggestion
    if "${kind}" get clusters | grep -q "^${cluster}$"; then
```
For shellcheck? (same for other calls)